yarn-project/simulator/src/avm/journal/journal.test.ts

```diff
@@ -102,15 +102,15 @@ describe('journal', () => {
     journal.writeL1Message(logs);
     journal.writeNullifier(commitment);
 
-    const childJournal = new AvmWorldStateJournal(journal.hostStorage, journal);
+    const childJournal = new AvmPersistableStateManager(journal.hostStorage, journal);
     childJournal.writeStorage(contractAddress, key, valueT1);
     await childJournal.readStorage(contractAddress, key);
     childJournal.writeNoteHash(commitmentT1);
     childJournal.writeLog(logsT1);
     childJournal.writeL1Message(logsT1);
     childJournal.writeNullifier(commitmentT1);
 
-    journal.acceptNestedWorldState(childJournal);
+    journal.acceptNestedCallState(childJournal);
```

Comment:
this is not always a nested call right? in a block it can be an entire transaction

Reply:
Good point :thinking: Maybe we rename this `acceptChildState`, `acceptOtherStateModifications`, ...